public/stylesheets/site/2.0/10-types-groups.css

```diff
@@ -190,12 +190,20 @@ span.series .position + .divider:after {
 }
 
 p.kudos {
-  background: url("/images/imageset.png") no-repeat -111px -580px;
-  padding: 0.5em 0.5em 0.5em 60px;
+  padding: 0.5em;
   margin: 2em 0;
   min-height: 40px;
 }
 
+p.kudos::before {
+  content: "";
+  background: url("/images/imageset.png") no-repeat -111px -580px;
+  width: 55px;
+  height: 54px;
+  float: left;
+  margin-top: -0.5em;
+}
+
```

Comment:
Since we've fixed the image, this change should be removed and we should go back to the original code without the float, which was:

https://github.com/otwcode/otwarchive/blob/f5d1fac3a815472ae5c3346450c46eb55fd3b46c/public/stylesheets/site/2.0/10-types-groups.css#L192-L197